src/Tokenizer/TokensAnalyzer.php

```diff
@@ -604,6 +604,8 @@ private function findClassyElements(Tokens $tokens, $start)
             }
         }
 
+        ksort($elements);
```

Comment:
TODO: send this bug fix to 2.2 line as well

Reply:
what is the bug actually ?

Reply:
When there are anonymous classes within a class which also has methods before (or after) the anonymous class, the returned elements are not sorted.
However most fixers that use this method assume the returned `[index, [...]]` array is sorted on the index; low-to-high. When out of order some edge cases will throw the fixers off when they iterate over the returned value and insert/remove tokens.